src/Tribe/Integrations/Beaver_Builder.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Handles compatibility with Beaver Builder plugin.
+ *
+ * @package Tribe\Events\Integrations
+ * @since 5.0.2
+ */
+namespace Tribe\Events\Integrations;
+
+use Tribe\Events\Views\V2\Template\Page;
+
+/**
+ * Integrations with Beaver Builder plugin.
+ *
+ * @package Tribe\Events\Integrations
+ * @since TBD
+ */
+class Beaver_Builder {
+
+	/**
+	 * Hooks all the required methods for Beaver_Builder usage on our code.
+	 *
+	 * @since TBD
+	 */
+	public function hook() {
+		// Bail when not on V2
+		if ( ! tribe_events_views_v2_is_enabled() ) {
+			return;
+		}
+
+		add_filter( 'fl_theme_builder_before_render_content', [ $this, 'action_restore_post' ] );
+	}
+
+	/**
+	 * Restore main post for Beaver Builder plugin.
+	 *ww
+	 * @since TBD
+	 *
+	 * @param int $post_id Items to be excluded by WP Rocket.
+	 *
+	 * @return void
```

Comment:
We usually do not specify `@return void` in our code, any specific reason to do it here? Oversight?